components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3033,6 +3039,7 @@ where
         // to the old region will stay in the original map.
         let region_locks = {
             let mut pessimistic_locks = self.fsm.peer.txn_ext.pessimistic_locks.write();
+            info!("moving {} locks to new regions", pessimistic_locks.len());
```

Comment:
I think it's not sound without region id...

Reply:
Sorry, I missed it...